crates/red_knot_python_semantic/resources/mdtest/type_properties/is_equivalent_to.md

```diff
@@ -254,4 +254,8 @@ from knot_extensions import is_equivalent_to, static_assert
 static_assert(is_equivalent_to(int | Callable[[int | str], None], Callable[[str | int], None] | int))
 ```
 
+### Overloads
+
+TODO
```

Comment:
Not that it needs to be done in this PR, but just noting here as an aside that it's always possible to implement `is_equivalent_to` for complex cases as simply `x.is_subtype_of(y) && y.is_subtype_of(x)` rather than writing a full dedicated implementation of it.

Reply:
Yeah, makes sense. Let me check this on Monday, will probably opt for delegating it to subtype check if it turns out to be complicated.